Update Function.java

chomats opened the pull request:
Add other case for function TRUNC supported by Oracle
Trunc(Date) and Trunc(String - date formated).

chomats committed:
Update Function.java

Add other case for function TRUNC supported by Oracle
Trunc(Date) and Trunc(String - date formated).

thomasmueller commented:
Could you add some test cases please? It's enough if you just comment a few SQL statements and the expected results (the format see testSql.txt).

chomats committed:
Update Function.java

Add other case for function TRUNC supported by Oracle
Trunc(Date) and Trunc(String - date formated).

+ test TestFunctionTrunc.java

chomats committed:
add .gitignore

chomats committed:
Change help.csv

Add new cases for TRUNCATE | TRUNC function.

thomasmueller commented:
The tests you send did not contain any assertions. I wrote new tests, and merged the patch (see my commit).
